CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+# Contributing to the Synthetics Agent
+
+The Elastic Synthetics Agent is open source and we love to receive contributions from our community — you!
+
+There are many ways to contribute, from writing tutorials or blog posts,
+improving the documentation, submitting bug reports and feature requests or
+writing code.
+
+You can get in touch with us through [Discuss](https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics),
```

Comment:
A comma doesn't make sense here. How about an [em dash](https://www.dictionary.com/e/em-dash/#:~:text=The%20em%20dash%20is%20an,or%20clause%20and%20adds%20emphasis.)?
```suggestion
You can get in touch with us through [Discuss](https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics)—
```